machine/machine_core/machine.py

```diff
@@ -300,7 +300,7 @@ def dhcp_server(self, mac='52:54:01', range=['10.111.112.2', '10.111.127.254']):
         cmd = "dnsmasq --domain=cockpit.lan " \
               "--interface=\"$(grep -l '{mac}' /sys/class/net/*/address | cut -d / -f 5)\"" \
               " --bind-interfaces --dhcp-range=" + ','.join(range) + ",4h" + \
-              " && firewall-cmd --add-service=dhcp"
+              "; systemctl start firewalld; firewall-cmd --add-service=dhcp"
```

Comment:
TIL

I always thought you needed `--wait` to do that.  Turns out waiting until startup is done is the (reasonable) default and you need to give `--no-block` if you don't like it.  `--wait` is something else entirely.